### PR TITLE
add jobs definition for upgrade b/w 4.12 stable build

### PR DESCRIPTION
**Pull request by rioliu-rh**
per discussion, I created job to test cert rotation with upgrade b/w 4.12 stable build. e.g. 4.12.22 to 4.12.23

/assign @jhou1 @liangxia

### CODE REVIEW

File: ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh [hunk 1]
```diff
@@ -107,7 +107,7 @@ oc adm ocp-certificates regenerate-leaf -n openshift-config-managed secrets kube
 
 # distribute trust across all known clients
 # update our local CA bundle so that when new serving certs are used for kube-apiserver we will trust them
-oc config refresh-ca-bundle
+oc config refresh-ca-bundle || true
```
Comment (wangke19) [inline ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh:110]: Try 
```
oc config refresh-ca-bundle || true
sleep 2
oc config refresh-ca-bundle
```

**Comment by liangxia** [comment]
Try to run `make update` locally and add those changes made by `make update`.

**Comment by liangxia** [comment]
/hold

Those jobs are for temp tests, no need to merge

File: ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh (hunk 1 above)
Comment (rioliu-rh) [reply ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh:110]: done

**Comment by rioliu-rh** [comment]
> Try to run `make update` locally and add those changes made by `make update`.

done

File: ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh (hunk 1 above)
Comment (rioliu-rh) [reply ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh:110]: rollback the change according to Ke's comment

**Comment by jianzhangbjz** [comment]
test for PR syncing.
/unhold

**Comment by liangxia** [comment]
PR sync issue is caused by github degrade, https://www.githubstatus.com/
You can open a new PR to workaround the issue.

**Pull request closed without merging**